From ddd9c439f942b199ba35daf82418d8f2f688d03a Mon Sep 17 00:00:00 2001
From: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
Date: Wed, 22 Jul 2020 18:03:55 +0530
Subject: [PATCH 8/8] SATA signal parameters can be tuned

For now we just keep Freescale's default values, but add Device Tree
properties to show how it's done. A list of permitted values can be found
in drivers/ata/ahci_imx.c

Signed-off-by: Antony Abee Prakash XV <antonyabee.prakashxv@adlinktech.com>
---
 arch/arm/boot/dts/lec-imx6q.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/lec-imx6q.dts b/arch/arm/boot/dts/lec-imx6q.dts
index b13412c221a3..fb0d1377df95 100644
--- a/arch/arm/boot/dts/lec-imx6q.dts
+++ b/arch/arm/boot/dts/lec-imx6q.dts
@@ -42,6 +42,11 @@
 };
 
 &sata {
+	/* Freescale's defaults */
+	fsl,transmit-level-mV = <1025>;
+	fsl,transmit-boost-mdB = <3330>;
+	fsl,transmit-atten-16ths = <9>;
+	fsl,receive-eq-mdB = <3000>;
 	status = "okay";
 };
 
-- 
2.17.1

